### PR TITLE
Add some of the missing marbles

### LINKED ISSUE #1463: Missing marble diagrams post SVG change
⚠️ Diagram changes / new diagrams in 3.2.x won't be accepted in PNG form and will be blocked until #1390 is merged.

> (❓) denotes a diagram that might either be hard to graphically represent, or one that is maybe too similar to existing variants to introduce another diagram

### Flux diagrams
 - [x] `index` (see #1448)
 - `limitRate` (see #1449, ⛔️ ==> deferred, see if somebody contributes)
 - [x] `limitRequest` (❓) => done
 - [x] `onBackpressureBuffer(Duration, maxSize)`
 - [x] `sort`
 - [x] `switchOnFirst`

### Mono diagrams
 - [x] `sequenceEqual`
 - ~`cache(Duration, Duration, Duration)`(❓)~ => a bit complex

### Diagrams relevant to both `Flux` and `Mono`
 - [x] `doFinally`
 - [x] `doOnEach`
 - ~`handle` (❓)~ => will do if requested, a bit complex
 - ~`name` and `tag` (❓)~ => will do if requested

### PR DESCRIPTION
**Pull request by simonbasle**
Fixes #1463.
To be merged with rebase.

### CODE REVIEW

**Comment by simonbasle** [comment]
@smaldini @bsideup tip for review: the rich diff view of github renders SVGs ;)

**Comment by simonbasle** [comment]
cc @OlegDokuka for the `switchOnFirst` marble. cc @rstoyanchev for the `switchOnFirst` javadoc.

File: reactor-core/src/main/java/reactor/core/publisher/Flux.java [hunk 1]
```diff
@@ -7911,34 +7925,36 @@ public final void subscribe(Subscriber<? super T> actual) {
...
+	 * argument to the {@link BiFunction} and should generally be built upon with operators.
+	 * <p>
 	 * Note that the source might complete or error immediately instead of emitting,
 	 * in which case the {@link Signal} would be onComplete or onError. It is NOT
 	 * necessarily an onNext Signal, and must be checked accordingly.
-     *
-	 * For example, this operator could be used to define a form of routing, depending on
-	 * the first element (which could contain routing metadata for instance):
+     * <p>
+	 * For example, this operator could be used to define dynamic transformation that depends
```
Comment (bsideup) [inline reactor-core/src/main/java/reactor/core/publisher/Flux.java]: missing article?

**Comment by simonbasle** [comment]
@bsideup we'll go back to `switchOnFirst` marble if needed 🕐, cleaning up the commits and merging this PR for now.

**Pull request merged**